Formula/trace2html.rb

```diff
@@ -1,16 +1,15 @@
 class Trace2html < Formula
   desc "Utility from Google Trace Viewer to convert JSON traces to HTML"
-  homepage "https://github.com/google/trace-viewer"
-  url "https://github.com/google/trace-viewer/archive/2015-07-07.tar.gz"
-  version "2015-07-07"
-  sha256 "6125826d07869fbd634ef898a45df3cabf45e6bcf951f2c63e49f87ce6a0442a"
-  revision 1
+  homepage "https://github.com/catapult-project/catapult/tree/master/tracing"
+  url "https://github.com/catapult-project/catapult/archive/93d499d2f97f3c298a77dff1a55befdf7e97d2a4.tar.gz"
```

Comment:
This is problematic since generally formulae without versioned releases are not accepted and if no releases are planned it would probably be more suitable to set up a private tap for this software.